patch.py

```diff
@@ -6,23 +6,40 @@
 from model import Html
 from data import HeropediaData
 
-parser = argparse.ArgumentParser(description="This software formats a Dota2' changelog text into HTML.")
-parser.add_argument('--file', '-f', action='store', help="changelog to be formated", required = True, dest = 'file')
-parser.add_argument('--version', '-v', action='version', version='%(prog)s: v1.0 (Yasha)')
+
+parser = argparse.ArgumentParser(
+description="This software formats a Dota2' changelog text into HTML.")
+parser.add_argument('--file', '-f', action='store',
+help="changelog to be formated", required=True, dest='file')
+parser.add_argument('--template', '-t', action='store',
+help="base template to generate HTML", default='default',
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->